clash_royale_service.py

```diff
@@ -97,3 +102,44 @@ def get_clan_cards_rank(self, clan_tag, card_type_filter='all'):
         member_cards_ranked = self.sort_list_by_card_level(member_cards_ranked)
 
         return clan_info, member_cards_ranked
+    '''
+    River Race Player Activity History Functions
+    '''
+    def get_river_race_log(self, clan_tag: str):
+        return self.clash_royale_client.get_river_race_log(clan_tag)
+
+    def clan_river_race_history(self, clan_tag: str, past_weeks=4) -> Dict[str, player_historical_activity]:

```

Comment:
/nit inconsistent spacing for `past_weeks=4`, whereas function below you have `past_weeks = 4`

Also, missing type hints, so should be `past_weeks: int = 4`